css/index.css

```diff
@@ -1,3 +1,375 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
 
 
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+html {
+	box-sizing: border-box;
+	
+	align-content: center;
+	
+}
+.container {
+	margin: auto;
+	width: 1000px
+	
+	
+}
+@media( max-width: 800px ){
+	.container {
+		width: 800px;
+	}
+
+
+}
+body header {
+	padding: 10px 0px;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+	
+}
+body header nav {
+	/* padding: 10px 50px; */
+	display: flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+	opacity: .5;
+}
+body header nav p{
+	margin-left: 40px;
+}
+@media( max-width: 800px ){
+	body header {
+		flex-direction: column-reverse;
+		align-items: center;
+	}
+
+
+}
+
+@media( max-width: 800px ){
+	body header nav {
+		margin: 50px
+	}
+
+}
+@media( max-width: 800px ){
+	.top-image {
+		display: none;
+	}
+
+}
+@media( max-width: 800px ){
+	.top-text {
+		text-align: center;
+	}
```

Comment:
you have a chunk of code here with 3 media queries for 800px. really all of these can be in one media query of 800, you'll just stack the three elements you're changing in there (body header, body heaver nav, and .top-image)